daemon/logger/factory/factory.go

```diff
@@ -0,0 +1,57 @@
+package factory
```

Comment:
@LK4D4 can be, however I thought logically that makes sense to separate. want me to move?